examples/wms/wms.md

```diff
@@ -0,0 +1,149 @@
+
+
+The `layers` option is a comma-separated list of layers. If a WMS service has defined several layers, then a request for a map image can refer to more than one layer.
+
+For the example WMS server we're using, there is a `ne:ne_10m_admin_0_countries` WMS layer showing country landmasses and country names, and a `ne:ne_10m_admin_0_boundary_lines_land` WMS layer showing country boundaries. The WMS server will compose both layers in one image if we request both, separated with a comma:
+
+	var countriesAndBoundaries = L.tileLayer.wms('http://demo.opengeo.org/geoserver/ows?', {
+		layers: 'ne:ne_10m_admin_0_countries,ne:ne_10m_admin_0_boundary_lines_land'
+	}).addTo(map);
+
+Note this will request *one* image to the WMS server. This is different than creating a `L.TileLayer.WMS` for the countries, another one for the boundaries, and adding them both to the map. In the first case, there is one image request and it's the WMS server who decides how to compose (put on top of each other) the image. In the second case, there would be two image requests and it's the Leaflet webpage who decides how to compose them.
+
+If we combine this with the [layers control](/examples/layers-control.html), then we can build a simple map to see the difference:
+
+	var basemaps = {
+		Countries: L.tileLayer.wms('http://demo.opengeo.org/geoserver/ows?', {
+			layers: 'ne:ne_10m_admin_0_countries'
+		}),
+
+		Boundaries: L.tileLayer.wms('http://demo.opengeo.org/geoserver/ows?', {
+			layers: 'ne:ne_10m_admin_0_boundary_lines_land'
+		}),
+
+		'Countries, then boundaries': L.tileLayer.wms('http://demo.opengeo.org/geoserver/ows?', {
+			layers: 'ne:ne_10m_admin_0_countries,ne:ne_10m_admin_0_boundary_lines_land'
+		}),
+
+		'Boundaries, then countries': L.tileLayer.wms('http://demo.opengeo.org/geoserver/ows?', {
+			layers: 'ne:ne_10m_admin_0_boundary_lines_land,ne:ne_10m_admin_0_countries'
+		})
+	};
+
+	L.control.layers(basemaps).addTo(map);
+
+	basemaps.Countries.addTo(map);
+
+Change to the "Countries, then boundaries" option, so you can see the boundaries "on top" of the landmasses, but the WMS server is clever enough to display building labels on top of that. It's up to the WMS server how to compose layers when asked for many.
+
+{% include frame.html url="wms-example3.html" %}
+
+
+### Notes to GIS users of WMS services
+
+From a GIS point of view, WMS handling in Leaflet is quite basic. There's no `GetCapabilities` support, no legend support, and no `GetFeatureInfo` support.
+
+`L.TileLayer.WMS` has extra options, which can be found in [Leaflet's API documentation](http://leafletjs.com/reference.html#tilelayer-wms-options). Any option not described there will be passed to the WMS server in the `getImage` URLs.
+
+Also note that Leaflet only supports `CRS:3857`. If your WMS service doesn't serve images in that coordinate system, you might need to use [Proj4Leaflet](https://github.com/kartena/Proj4Leaflet) to use a different coordinate system in Leaflet.
```

Comment:
might want to link to a page explaining EPSG3857, for example http://spatialreference.org/ref/sr-org/6864/